### DISCUSSION TITLE
lsp,bash为什么没提过语言服务器相关功能

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [nshen/learn-neovim-lua#90] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/43649186/184456580-77858abf-827b-40e6-b0ee-2e37f3e6e84f.png)
没有相关语法检查 语义跳转等功能

**Comment by HUAHUAI23** [comment, marked as answer]
忘记装shellcheck了 我擦

**Reply by nshen** [reply]
很少写 bash 哈，装了 shellcheck 很好用吗

**Comment by HUAHUAI23** [comment]
还可以，就和vscode的shellcheck插件一样，提供语法检查，你的配置文件太顶了，做了好多解藕，尤其是uconfig，把键盘映射分离了开来，参考了很多你的配置，非常感谢



---原始邮件---
发件人: "Xin ***@***.***&gt;
发送时间: 2022年8月13日(周六) 中午11:53
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [nshen/learn-neovim-lua] lsp,bash为什么没提过语言服务器相关功能 (Discussion #90)




 
很少写 bash 哈，装了 shellcheck 很好用吗

**Reply by nshen** [reply]
还在瞎折腾哈

**Reply by HUAHUAI23** [reply]
哈哈
